benchmark/src/rest-routing/README.md

```diff
@@ -11,29 +11,39 @@ https://loopback.io/doc/en/lb4/Routing-requests.html for more information.
 
 ```sh
 npm run -s benchmark:routing // default to 1000 routes
-npm run -s benchmark:routing -- <number-of-routes>
+npm run -s benchmark:routing -- <number-of-routes> <number-of-routes>
+```
+
+For example:
+
+```
+npm run -s benchmark:routing -- 10 40 160 640 2560
 ```
 
 ## Base lines
 
 ```
 name                 duration    count    found   missed
-TrieRouter          0,1453883       10        8        2
-RegExpRouter        0,1220030       10        8        2
+TrieRouter          0,1038244       10        8        2
+RegExpRouter        0,1002559       10        8        2
+
 
 name                 duration    count    found   missed
-TrieRouter          0,2109957       40       35        5
-RegExpRouter        0,8762936       40       35        5
+TrieRouter           0,661415       40       35        5
+RegExpRouter        0,5797990       40       35        5
+
 
 name                 duration    count    found   missed
-TrieRouter          0,4895252      160      140       20
-RegExpRouter       0,52156699      160      140       20
+TrieRouter          0,2851976      160      140       20
+RegExpRouter       0,27040556      160      140       20
+
 
 name                 duration    count    found   missed
-TrieRouter         0,16065852      640      560       80
-RegExpRouter      0,304921026      640      560       80
+TrieRouter          0,8914005      640      560       80
+RegExpRouter      0,330599418      640      560       80
```

Comment:
Weird. If I am reading these numbers correctly, then TrieRouter is about 2.5 times slower than RegExpRouter in this scenario. Is that expected?

Reply:
No, TrieRouter takes `8,914,005ns (8.9ms)` while RegExpRouter takes `330,599,418ns (330ms)` to process 640 requests.